### ISSUE TITLE
data-uk-observe working only once for ajax

### ISSUE THREAD

**Issue by nitishkumar71** [uikit/uikit#1693]
Hello, I am new in UIKit.
I am using a <div> to load webpage. I am using JQuery  load() to load webpage.
Code for div is

```
<div id='my-id' data-uk-observe><div>
```

JQuery Code for loading webpage

```
$("#my-id").load("url");
```

My problem is that div is updating only once. If I try to load second page, div is showing blank.
Even though elements of second page are loaded within <div>. I debugged this via Inspect element option in browser

**Comment by philoest** [comment]
We have released the next major version, UIkit3. To start focusing our work on the most recent release, we decided to close all the issues related to UIkit2. Please check, if these issues still persist in version 3 and if so, please open a new issue. 

Thanks for your understanding,
Your UIkit Team
